src/components/LogTable.vue

```diff
@@ -107,19 +107,71 @@ export default {
               return;
             }
 
+
             switch (col.name) {
               case 'Log type':
                 where.type = { $iLike: `%${value}%` };
                 break;
               case 'App':
                 where.app = { name: { $iLike: `%${value}%` } };
                 break;
-              case 'User':
-                where.$or = [
-                  { user: { type: null, email: { $iLike: `%${value}%` } } },
-                  { data: { _userEmail: { $iLike: `%${value}%` } } }
-                ];
+
+              case 'User': {
+                const stringRegExp = '([-a-zA-Z0-9@:%_\\+.~#?&//=]*)';
```

Comment:
"stringRegExp" is quite an abstract name. Can you rename and/or add a comment to outline what this regexp does?

Reply:
I started with matches for email only. However, when I tried string `as aneri` then it was not showing a result as `aneri` is not email. As per the user experience, any string having alphabets, numeric and special characters should be filtered. Therefore, added a simple string regular expression.